src/EdFi.AnalyticsMiddleTier.Tests/.env.example

```diff
@@ -1,12 +1,13 @@
 USE_MSSQL_DEFAULT_CONN_STRING=TRUE
 USE_POSTGRES_DEFAULT_CONN_STRING=TRUE
-SQLSERVER_SERVER=localhost
-SQLSERVER_SA_PASS=123
 SQLSERVER_DATABASE_DS2=AnalyticsMiddleTier_Testing_Ds2
 SQLSERVER_DATABASE_DS31=AnalyticsMiddleTier_Testing_Ds31
 SQLSERVER_DATABASE_DS32=AnalyticsMiddleTier_Testing_Ds32
 SQLSERVER_DATABASE_DS33=AnalyticsMiddleTier_Testing_Ds33
 SQLSERVER_INTEGRATED_SECURITY=false
+SQLSERVER_SERVER=localhost
+SQLSERVER_ADMIN_USER=sa
+SQLSERVER_ADMIN_PASS=123
 SQLSERVER_USER=ods_user
 SQLSERVER_PASS=123
 POSTGRES_HOST=localhost
```

Comment:
I know it's not part of the scope of this ticket, but maybe in this file `USE_MSSQL_DEFAULT_CONN_STRING` and `USE_POSTGRES_DEFAULT_CONN_STRING `should be `False`.
This because, once the user copies this file to create the `.env` file, we can assume he is doing it because he wants to use the parameters for the connection string instead of the default connection string; in such a case, these values should be false. Makes sense?